tasks/eva_3145/src/main/groovy/eva3145/eva3145_analysis.groovy

```diff
@@ -0,0 +1,193 @@
+package eva3145
+
+import eva3145.AsmMatchMetrics
+import groovy.cli.picocli.CliBuilder
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import uk.ac.ebi.ampt2d.commons.accession.core.models.EventType
+import uk.ac.ebi.eva.accession.core.GenericApplication
+import uk.ac.ebi.eva.accession.core.model.dbsnp.DbsnpSubmittedVariantEntity
+import uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment
+import uk.ac.ebi.eva.groovy.commons.EVAObjectModelUtils
+import uk.ac.ebi.eva.groovy.commons.RetryableBatchingCursor
+
+import static org.springframework.data.mongodb.core.query.Query.query
+import static uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment.*
+import static org.springframework.data.mongodb.core.query.Criteria.where
+
+// This script analyzes various metrics about variants with assembly match - see EVA-3145
```

Comment:
Structure of the script adapted from [here](https://github.com/EBIvariation/eva-tasks/blob/master/tasks/eva_3097/src/main/groovy/eva3097/eva3097_analysis.groovy)